src/evaluation/Run.jl

```diff
@@ -196,6 +196,47 @@ update_save_run!(session::ServerSession, notebook::Notebook, cell::Cell; kwargs.
 update_run!(args...) = update_save_run!(args...; save=false)
 
 
+function update_from_file(session::ServerSession, notebook::Notebook)
```

Comment:
@pankgeorg get this to main, after you add tests

Reply:
`update_from_file` may change `session` and `notebook`. Because it executes `update_save_run!` at last line. I think that `update_from_file!` is better name than it.

Reply:
@p1scescom can you make a PR? It's already merged and released, so use `@deprecate`